pkg/cmd/gist/view/view.go

```diff
@@ -137,6 +139,13 @@ func viewRun(opts *ViewOptions) error {
 	}
 	sort.Strings(filenames)
 
+	if opts.Files {
+		for _, fn := range filenames {
+			fmt.Fprintf(opts.IO.Out, "%s\n", cs.Gray(fn))
```

Comment:
I say we don't use any color for this listing. It can simply be
```suggestion
			fmt.Fprintln(opts.IO.Out, fn)
```